.github/workflows/codecov_ats.yml

```diff
@@ -80,13 +87,43 @@ jobs:
           env:
             CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
             CODECOV_STATIC_TOKEN: ${{ secrets.CODECOV_STATIC_TOKEN }}
-        # Run label analysis IN DRY MODE (no tests will actually run)
-        - name: Label Analysis
+        # Run Automated Test Selection in dry mode to get the list of tests to run
+        # The base commit will be the parent commit (apparently commits on master don't exist in codecov)
+        - name: Codecov Automated Test Selection
+          id: codecov_automated_test_selection
           continue-on-error: true
```

Comment:
To avoid the annoying ❌ in the developer's commit, only reason.
This step is the most likely to fail.
Same reason this job is named "IGNORE codecov ats"